host_v2/src/components/TitleCCSS.tsx

```diff
@@ -0,0 +1,43 @@
+
+import React from 'react';
+import { Typography } from '@mui/material';
+import { styled } from '@mui/material/styles'; // Import styled from MUI
```

Comment:
totally valid comment but because we use it throughout the components, I feel like we can drop it here

Reply:
gone